### DISCUSSION TITLE
Ristrutturazione Casa

### DISCUSSION THREAD

**Discussion by 19melos87** [indomus/forum#1164] [GENERALE - Discussioni generiche]
Ciao a tutti,

come da titolo, mi appresto a ristrutturare casa. Non dovrò fare grandi lavori ma visto che devo imbiancare tutto sto valutando l'eventualità di fare anche lavori sulle prese, sugli interruttori ed altro.

Naturalmente vorrei progettare bene il tutto dall'inizio e quindi vorrei cominciare a decidere intanto su cosa installare HA.
Sto pensando di assemblarmi un mini pc dato che oltre ad HA ho necessità di mettere altre cose (prima tra tutte PLEX). Voi cosa mi consigliate? Al momento plex ce l'ho su un vecchio NAS (zyxel325v2) che sicuramente prima o poi braserò mettendoci su qualche firmware moddato.

Poi per quanto riguarda il resto, valutando il fatto che forse andrò a cambiare anche tutte le placchette delle 503 de che qualche lavoretto dovrò farlo voi consigliate un impianto wired per quanto riguarda la parte domotica oppure vado su zigbee?

Grazie intanto per chi mi aiuterà in questo mio lungo progetto, man mano che mi verranno delle domande aggiornerò il post!

**Comment by ocimin** [comment]
Io ho fatto installazione su un minipc (un assemblato usando una vecchia mobo mini-itx) e mi trovo bene. Le mie necessità erano di avere un computer abbastanza potente da poterci installare un linux da usare come server per gestire diversi servizi (vpn, HA, file server, transmission, etc..) e da poter collegare più dischi rigidi per poter fare un raid, il tutto consumando il meno possibile considerando che deve stare acceso 24/7 (mediamente il mio assorbe 30watt). In pratica un NAS un po' più pompato e versatile. Sto pensando anche di usarlo come media center (io uso kodi) in quanto gli ho montato un alimentatore fanless e quindi potrebbe stare tranquillamente in salotto senza dare fastidio, ma non ho ancora avuto voglia/tempo. Il vantaggio rispetto ad un NAS è che lo modelli davvero come vuoi, ma questo è anche il rovescio della medaglia in quanto è più facile che ci siano casini da risolvere ad esempio in fase di configurazione dei servizi o quando fai aggiornamenti. Per la connessione al minipc io preferisco di gran lunga quella cablata, sia per le prestazioni che per la stabilità, ma non mi spaccherei la testa se fosse troppo complesso tirare i cavi: se non hai logistiche particolarmente sfavorevoli i nuovi protocolli wifi soprattutto in ambiente domestico sono altamente performanti.

**Comment by 19melos87** [comment]
Sto pensando di usare questo vecchio pc, ma vorrei capire se il rapporto consumi/prestazioni vale la pena... Naturalmente ci dovrebbe girare come ho scritto sia HA che Plex, ma non escludo in futuro di poterlo usare per altro.
Il case è questo che vedete in foto (un cooler master Case Elite 110)
[spoiler]
(![1](https://user-images.githubusercontent.com/12113379/146183150-e2ecdb82-be5f-4bad-84cd-2633c6bf4629.jpeg)
![2](https://user-images.githubusercontent.com/12113379/146183154-a03e9822-e31e-413a-a044-defd1661bbd7.jpeg)
![3](https://user-images.githubusercontent.com/12113379/146183157-baec0787-c8a4-4b95-87e1-a316adf2ca8f.jpeg)
[/spoiler]

Dentro si presenta così:
![4](https://user-images.githubusercontent.com/12113379/146183166-fc517a8d-d038-40c6-8503-fcf273ef5d54.jpeg)

Come cpu monta un Intel i3-4170 3.70 ghz.

Le ram sono 2 banchi DDR3 da 4gb come da foto
![5](https://user-images.githubusercontent.com/12113379/146183171-c5c60592-c9b7-4a08-a7da-3b54db43fba5.jpeg)

Che ne pensate?

**Comment by nromagna** [comment]
Ma adottare una soluzione tipo:
Raspberry Pi solo HA e un nas per Plex tipo Synology? Nel caso opti per il nas potresti non prendere Raspberry ed installare tutto su NAS.
Certo i prezzi salgono rispetto al tuo pc, però...

**Comment by 19melos87** [comment]
Credo che per ora opterò per l'uso di questo pc visto che ce l'ho e se devo
spendere andrò a spendere su quello che mi manca per domotizzare.

Il giorno mer 15 dic 2021 alle ore 13:47 nromagna ***@***.***>
ha scritto:
